src/js/components/Grid/StyledGrid.js

```diff
@@ -213,12 +212,7 @@ const areasStyle = (props) => {
     .join(' ')};`;
 };
 
-const StyledGrid = styled.div
-  .withConfig(styledComponentsConfig)
-  .attrs((props) => ({
-    ...withTheme(props),
-    'aria-label': props.a11yTitleProp,
```

Comment:
do we need to add this or its fine being removed? not sure why it was pulled out but we no longer have `aria-label` in this file just making sure that is intentional?

Reply:
This change is intentional, in the Grid.js file I renamed a11yTitleProp to aria-label so that it will pass through correctly without needing `'aria-label': props.a11yTitleProp` in attrs.